examples/babylon-es5.html

```diff
@@ -23,10 +23,9 @@
 		<script type="module">
 			import { LookingGlassWebXRPolyfill } from "../dist/bundle/webxr.js"
 	          new LookingGlassWebXRPolyfill({
-                    tileHeight: 512,
```

Comment:
I know a customer just hit an issue with this - are the examples and documentation updated to reflect that we no longer use these properties?

Given this is a backwards compatibility breaking change, we should consider doing a major increment on the SemVer. Though I guess being in 0.x means we have more freedom there.

Reply:
this was made in a prior release

I'll update the docs today.